### DISCUSSION TITLE
Readthedocs improvements

### DISCUSSION THREAD

**Discussion by bartvstratum** [microhh/microhh#91] [Ideas]
Please list anything about `microhh.readthedocs.io` that is missing, incomplete, not described in enough detail, etc.

**Comment by bartvstratum** [comment]
The `.ini` file overview of options is incomplete. I will start working on that.

**Reply by bartvstratum** [reply]
Should be up-to-date with the latest `main` now!

**Comment by bartvstratum** [comment]
Add a section about contributions to the code; use a fork, commit, make pull request, etc.

**Reply by bartvstratum** [reply]
This is briefly documented in the `README.md` on Github: https://github.com/microhh/microhh#contributing

**Comment by WBMol** [comment]
The large scale subsidence can be turned on via the `swwls` option in `[Force]` [see docs](https://microhh.readthedocs.io/en/latest/setting_up_a_case/the_ini_file.html#large-scale-forcings-force), but it's not explained how. Setting to 1 fails.  Based on  `/src/force.cxx`, it appears there are two options: 'mean' and 'local'. I can not find any further explanation. 

The section that describes how to setup the [input file](https://microhh.readthedocs.io/en/latest/setting_up_a_case/the_input_file.html) uses `swwls=1`.

**Reply by bartvstratum** [reply]
I ran into the same problem on Friday, I'll fix it!
